### PR TITLE
V 1.1.0

### PR DESCRIPTION
**Pull request by marcus-j-davies**
Fixes : https://github.com/NRCHKB/node-red-contrib-unifi-os/issues/73
Fixes  : https://github.com/NRCHKB/node-red-contrib-unifi-os/issues/72
Fixes : https://github.com/NRCHKB/node-red-contrib-unifi-os/issues/51

 - Adds Line Crossing event support
 - Improves  websocket management

### CODE REVIEW

**Comment by Shaquu** [comment]
That is a lot of lines

**Review by jurajs5 — APPROVED** [review]

**Comment by marcus-j-davies** [comment]
1 problem (on the Unifi end) I have discovered 
If multiple Lines are added to a camera, it does not show the state of them  - they are still In the payload, but not their state.

When the camera has 1 Line :
`payload.metadata.linesStatus` exists 

When the camera has > 1 Line(s) :
`payload.metadata.linesStatus` does not exists 

Not sure we can fix that, as its a problem, in the original payload form Unifi

🤷

**Comment by crxporter** [comment]
**Push it!**
**Push it *real good!***

**Comment by crxporter** [comment]
I don't even see anywhere on the official webpage or app where it will show *which* line was crossed...

**Review by crxporter — APPROVED** [review]

**Review by crxporter — DISMISSED** [review]

**Review by jurajs5 — APPROVED** [review]

**Comment by jurajs5** [comment]
Something is better than nothing - push it. Hope in some new updates, we will get info which one was crossed.

**Comment by marcus-j-davies** [comment]
Beta available.

within the `.node-red` dir.
```
npm install node-red-contrib-unifi-os@dev
```

**Comment by marcus-j-davies** [comment]
@Supergiovane 

Has your system been stable since your contriubtion (that I have merged into this PR)?
I want to do a release maybe today - providing you have not needed to address any issues since?

**Comment by Supergiovane** [comment]
> @Supergiovane
> 
> Has your system been stable since your contriubtion (that I have merged into this PR)? I want to do a release maybe today - providing you have not needed to address any issues since?

Hi
Yes, rock solid.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "node-red-contrib-unifi-os",
-    "version": "1.0.1",
+    "version": "1.1.0-beta.3",
```
Comment (Shaquu) [inline package.json:3]: Version change before merge?

**Review by Shaquu — DISMISSED** [review]

**Pull request merged**